timescaledb/how-to-guides/migrate-data/migrate-mst-cloud.md

```diff
@@ -0,0 +1,478 @@
+# Migrate from Managed Service for TimescaleDB to Timescale Cloud
+
+Migrate your data from Managed Service for TimescaleDB to Timescale Cloud to take
+advantage of Cloud's features.
+These include separate scaling for compute and storage requirements, first-class 
+multi-node support, and more.
+
+There are two methods for migrating your data:
+
+- [**Migrate the entire database at once using the PostgreSQL utilities `pg_dump` 
+  and `pg_restore`.**](#migrate-the-entire-database-at-once)
+  Choose this option for smaller databases.
+  This method directly transfers all data and schemas, including TimescaleDB-specific 
+  features.
+  Your hypertables, continuous aggregates, and policies are automatically
+  available in Cloud.
+
+- [**Migrate the schema, and then migrate the data separately.**](#migrate-the-schema-and-data-separately)
+  Choose this option for larger databases.
+  This method allows you to individually migrate tables and chunks.
+  If the migration fails mid-way, you can restart from the failure point rather
+  than the beginning. With this method, TimescaleDB-specific features won't be 
+  migrated.
+  See the instructions below to restore your hypertables, continuous
+  aggregates, and policies. 
+
+<highlight type="tip">
+The recommended migration method depends on both your database size and your network
+upload and download speeds.
+If you aren't sure which method to use, start with the first method and try running
+`pg_dump` on one table. 
+If it takes too long, switch to the second method.
+</highlight>
+
+## Migrate the entire database at once
+
+Migrate smaller databases by using `pg_dump` and `pg_restore` on your entire 
+database.
+
+<highlight type="note">
+Depending on the size of your database and your network speed, migration may take 
+several hours.
+You can continue reading and writing(?) to your old database during this time.
```

Comment:
@lukas-bernert Could you clarify what is safe to do here please? I remember reading is fine, but does writing work? Is there a scenario where data could be lost if writes occur to a part of the database that was already copied, or would that never happen?

And for bonus information: what is safe to do in the long method while `\COPY` is happening?

Reply:
it does affect performance. see https://stackoverflow.com/a/50129088
So if users have a heavy ingest process going on they might want to scale up their service config if they think performance will become a problem.
However, for strategy 2 pg_dump is only used for schema things which should be very light weight.

This only applies to pg_dump in strategy 1 or the \COPY command in strategy 2.

Moreover, I think pg_dump or \COPY will export data as it was at the time of running the command. So if the user updated data after the command has been run, those changes won't be reflected in the data export.
@feikesteenbergen can you confirm?